addons/web_editor/static/src/js/backend/html_field.js

```diff
@@ -386,14 +479,18 @@ export class HtmlField extends Component {
         const $editable = this.wysiwyg.getEditable();
         const html = this.wysiwyg.getValue();
         const $odooEditor = $editable.closest('.odoo-editor-editable');
+        // Save correct nodes references.
+        const fragmentBackup = document.createDocumentFragment();
```

Comment:
I suggest renaming this variable to `originalContents` or `initialContents` or something else along those lines if you have other ideas.